R/coxphSGD-GEN.R

```diff
@@ -1,21 +1,21 @@
-#' Stochastic Gradient Descent log-likelihood estimation in 
+#' Stochastic Gradient Descent log-likelihood estimation in
```

Comment:
@zzawadz this whole file is generated automatically since I have wrote whole (code, documentation and examples) package in the vignette